glib/src/auto/functions.rs

```diff
@@ -701,25 +701,22 @@ pub fn spaced_primes_closest(num: u32) -> u32 {
 }
 
 #[doc(alias = "g_spawn_async")]
-pub fn spawn_async(
+pub fn spawn_async<P: FnOnce() + Send + Sync + 'static>(
     working_directory: Option<impl AsRef<std::path::Path>>,
     argv: &[&std::path::Path],
     envp: &[&std::path::Path],
     flags: SpawnFlags,
-    child_setup: Option<Box_<dyn FnOnce() + 'static>>,
+    child_setup: P,
```

Comment:
This is wrong. The callback is optional :)

Reply:
https://gitlab.gnome.org/GNOME/glib/-/merge_requests/3523

Reply:
This change still didn't land in the gir-file...

Reply:
Need to send an MR to gobject-introspection to update its glib snapshot

Reply:
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/420

Reply:
Fixed